content/src/main/content/jcr_root/apps/acs-commons/components/utilities/manage-controlled-processes/clientlibs/js/scriptRunner.js

```diff
@@ -109,17 +109,20 @@ var ScriptRunner = {
         });
     },
     startProcess: function () {
-        var data = {};
-        jQuery("#processDefinitionInput form", window.top.document).serializeArray().map(function (x) {
-            data[x.name] = x.value;
-        });
+        /* Use FormData object to support file uploads */
```

Comment:
@badvision - just want to make sure you saw this change as well; I didn't quite grok why you were doing the map to the data object.